src/shared/components/results/__tests__/Results.spec.tsx

```diff
@@ -4,9 +4,20 @@ import Results from '../Results';
 
 import customRender from '../../../__test-helpers__/customRender';
 
-import '../../../../uniprotkb/components/__mocks__/mockApi';
 import { UniProtKBColumn } from '../../../../uniprotkb/types/columnTypes';
 
+import '../../../../uniprotkb/components/__mocks__/mockApi';
+
+jest.mock('../SearchSuggestions', () => ({
+  __esModule: true,
+  default: () => '{{ SearchSuggestions }}',
+}));
+
+jest.mock('../DidYouMean', () => ({
+  __esModule: true,
+  default: () => '{{ DidYouMean }}',
+}));
+
```

Comment:
Same thing as before when mocking components, but here we don't have snapshots at the moment, but at least we don't trigger extra related logic that is already tested somewhere else => quicker test here, less things to mock, and less things that might break